modules/gitops-release-notes-1-5-0.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-5-0_{context}"]
+= Release notes for {gitops-title} 1.5.0
+
+[role="_abstract"]
+{gitops-title} 1.5.0 is now available on {product-title} 4.7, 4.8, 4.9 and 4.10.
```

Comment:
@wtam2018 OCP 4.7 support is been removed right?

Reply:
right